src/search/html.ml

```diff
@@ -16,7 +16,9 @@ let url { Entry.id; kind; doc = _ } =
 
        Values, types, ... are not sensitive to [stop_before], allowing us to
        shorten the match. *)
-    match kind with Doc _ -> false | _ -> true
+    match kind with
+    | Doc _ -> false
+    | _ -> true
```

Comment:
Are match always on multiple lines now?

Reply:
This is due to the comment being on its own line:
```ocaml
let _ =
  (* foo *)
  match kind with
  | Doc _ -> false
  | _ -> true

let _ = (* foo *) match kind with Doc _ -> false | _ -> true
```
I think it makes sense, if this match is important enough to require a comment, it should also be made more readable and editable.

Reply:
Makes a lot of sense. The comment was so big that I did not notice it.